plugins/integrations/prometheus/src/main/java/org/apache/cloudstack/metrics/PrometheusExporterImpl.java

```diff
@@ -158,20 +158,20 @@ private void addHostMetrics(final List<Item> metricsList, final long dcId, final
             final String cpuFactor = String.valueOf(CapacityManager.CpuOverprovisioningFactor.valueIn(host.getClusterId()));
             final CapacityVO cpuCapacity = capacityDao.findByHostIdType(host.getId(), Capacity.CAPACITY_TYPE_CPU);
             if (cpuCapacity != null && cpuCapacity.getCapacityState() == CapacityState.Enabled) {
-                metricsList.add(new ItemHostCpu(zoneName, zoneUuid, host.getName(), host.getUuid(), host.getPrivateIpAddress(), cpuFactor, USED, cpuCapacity.getUsedCapacity(), isDedicated, hostTags));
+                metricsList.add(new ItemHostCpu(zoneName, zoneUuid, host.getName(), host.getUuid(), host.getPrivateIpAddress(), cpuFactor, ALLOCATED, cpuCapacity.getUsedCapacity(), isDedicated, hostTags));
```

Comment:
Usually, Allocated space is the sum of used and reserved capacity. I'm not sure if getting "usedCapacity" and showing it as "Allocated" is the right way.

@soreana Regarding the differences you are mentioning in the issue, the "memoryused" from the hostreponse is from the [hoststats](https://github.com/apache/cloudstack/blob/6125886f3d0b64ff3d0a743d00bf414774e7e2e3/server/src/main/java/com/cloud/api/query/dao/HostJoinDaoImpl.java#L227) which is collected by stats collector from host and "cloudstack_host_memory_usage_mibs_total" from Prometheus is the value from "op_host_capacity" table (cpuCapacity.getUsedCapacity()). 

If we want to show the same metrics in both hostmetrics and Prometheus then we need to get the usedcapacity from hoststats and report it as "USED" only.

Reply:
The reserved capacity subtracted from the total memory of the host. I've checked that in a refresh test environment and the sum of the VM memories was equal to the usedCapacity even though I've changed the reserved memory.

Thanks for sharing the info, I would add a new entry for the used. :+1:

Reply:
Thanks @soreana

Reply:
@harikrishna-patnala I've added the used labels as well.